csrankings-k.csv

```diff
@@ -59,6 +59,7 @@ Kai Shen,University of Rochester,https://www.cs.rochester.edu/u/kshen,NOSCHOLARP
 Kai Shu,Illinois Institute of Technology,http://www.cs.iit.edu/~kshu,-6bAV2cAAAAJ
 Kai Shuang,BUPT,https://teacher.bupt.edu.cn/shuangkai/en/index.htm,KjjEavgAAAAJ
 Kai Sun 0001,University of Tennessee,http://web.eecs.utk.edu/~kaisun,O_2drywAAAAJ
+Kai Wang 0037,Shanghai Jiao Tong University,https://cs-kaiwang.github.io/,G4DiwTIAAAAJ

```

Comment:
Not in CS, does not appear to meet inclusion criteria. Please remove or justify as directed by the instructions.

Reply:
Dear Prof. Berger,

Wang Kai belongs to the Department of Data and Business Intelligence. This department primarily focuses on issues related to computer science and endeavors to utilize computer science to solve business applications. All of Wang Kai's research papers are published in computer science conferences and journals, such as ICDE, VLDB, SIGMOD, among others. For further validation, please refer to Wang Kai's Google Scholar profile ([https://scholar.google.com/citations?user=G4DiwTIAAAAJ&hl=zh-CN](https://scholar.google.com/citations?user=G4DiwTIAAAAJ&hl=zh-CN)). Please reconsider this application.

Regards,
Shunyang & Kai Wang

Reply:
This does not at all speak to the inclusion criteria: can or can they not _solely_ supervise a PhD _in Computer Science_?

Reply:
Dear Prof. Berger,

Thanks for your reply.

Yes, he is PhD advisor right now, and he can solely supervise PhD in computer science, as pointed out on the home page: [https://cs-kaiwang.github.io/](https://cs-kaiwang.github.io/). In addition, the students he supervised right now all pursue publications in the field of database and data management (e.g., SIGMOD, VLDB, ICDE).

Thanks for your assistance.

Regards,
Shunyang & Kai Wang

Reply:
The inclusion criteria is that the PhD must be _in Computer Science_ and not in "related" fields. The home page does not clarify anything. I note that he is not listed here: https://www.cs.sjtu.edu.cn/en/W.aspx. The fact that a person has published in CS conferences is not actually relevant to this question.